content/docs/2.9/scalers/apache-kafka.md

```diff
@@ -47,6 +48,7 @@ partitions on a topic, allowing for idle consumers. (Default: `false`, Optional)
 - `scaleToZeroOnInvalidOffset` - This parameter controls what the scaler does when a partition doesn't have a valid offset.
 If 'false' (the default), the scaler will keep a single consumer for that partition. Otherwise ('true'), the consumers for that
 partition will be scaled to zero. See the [discussion](https://github.com/kedacore/keda/issues/2612) about this parameter.
+- `excludePersistentLag` - When set to `true`, the scaler will exclude partition lag for partitions which current offset is the same as the current offset of the previous polling cycle. This parameter is useful to prevent scaling due to partitions which current offset message is unable to be consumed.
```

Comment:
Can you please document the default behavior as per https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler

Reply:
Added documentation for default behavior